src/app/GlobalStyle.tsx

```diff
@@ -31,7 +31,8 @@ const GlobalStyle = createGlobalStyle`
 
   body {
     padding: 0;
-    color: var(--c-dark); 
+    color: var(--c-dark);
+    background-color: var(--c-gray-100) ;
```

Comment:
I'd think about renaming your global colors. It's not good practice to name the variables with the actual color because in case you want to change the colors in globals, you won't only need to change the color itself but also the names (including in all files where you use them).